src/handlers.js

```diff
@@ -234,9 +237,9 @@ const areValidTypes = function(playerPowerplants, selectedResourceDetails) {
   const storageCapacity = getStorageCapacity(playerPowerplants);
   const selectedResources = parseResourceDetails(selectedResourceDetails);
   const selectedResourceTypes = Object.keys(selectedResources);
-  if (storageCapacity["Hybrid"]) {
-    storageCapacity["Coal"] += storageCapacity["Hybrid"];
-    storageCapacity["Oil"] += storageCapacity["Hybrid"];
+  if (storageCapacity[RESOURCE_TYPE.HYBRID]) {
```

Comment:
I specially asked for these things... destructure these so that will make more sense